kafkautils/producer.go

```diff
@@ -23,6 +23,7 @@ type ProducerInterface interface {
 	EnableTransactions() error
 	WriteAndCommitSink(<-chan *Message)
 	WriteAndCommit(Topic, []byte, proto.Message) error
+	MakeQueueTransactionSink() chan *Message
```

Comment:
Isn't it duplicate to `WriteAndCommitSink(<-chan *Message)`?

Reply:
It's not duplicate. It uses `WriteAndCommitSink`

Reply:
If it's a wrapper function, maybe we don't need to put it in the interface, what do you think?

Reply:
I know. I just doubt if it's a good idea to just add a wrapper function to the interface cuz it's growing bigger and bigger and this is the library all connectors will depend on in the future.